packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -764,88 +729,39 @@ abstract class BaseFlutterTask extends DefaultTask {
 
         intermediateDir.mkdirs()
 
```

Comment:
Its probably easier to look side-by-side on this one. Essentially we make only a single call into flutter now, using the assemble API instead.